x/gov/client/cli/query.go

```diff
@@ -251,26 +350,43 @@ func GetCmdQueryDeposits(queryRoute string, cdc *codec.Codec) *cobra.Command {
 		},
 	}
 
-	cmd.Flags().String(flagProposalID, "", "proposalID of which proposal's deposits are being queried")
-
 	return cmd
 }
 
 // GetCmdQueryTally implements the command to query for proposal tally result.
 func GetCmdQueryTally(queryRoute string, cdc *codec.Codec) *cobra.Command {
 	cmd := &cobra.Command{
-		Use:   "tally",
+		Use:   "tally [proposal-id]",
+		Args:  cobra.ExactArgs(1),
 		Short: "Get the tally of a proposal vote",
+		Long: strings.TrimSpace(`
+Query tally of votes on a proposal. You can find the proposal-id by running gaiacli query gov proposals:
+
+$ gaiacli query gov tally 1
```

Comment:
ditto

Reply:
going to mark these as resolved as the point of this PR is to improve UX and prevent users from having to type in multiple ~10 character flags to do so.